src/settings.h

```diff
@@ -10,6 +10,9 @@
 
 #include <QColor>
 #include <QObject>
+#include <QVector>
+
+#include "models/data.h"
```

Comment:
forward declare the Parameters struct instead please

Reply:
not possible, but I moved the struct to settings.h